src/components/card/SourceCard.vue

```diff
@@ -0,0 +1,66 @@
+<template>
```

Comment:
You can use a Fieldset with a card as slot. Maybe we can create a FieldsetCard component.